finesse/gui/temp_control.py

```diff
@@ -181,46 +183,52 @@ def _create_controls(self) -> QGridLayout:
         return layout
 
 
-class TC4820_HOT(QGroupBox):
-    """Widgets to view the TC4820 HOT properties."""
+class TC4820(QGroupBox):
+    """Widgets to view the TC4820 properties."""
 
-    def __init__(self) -> None:
-        """Creates the widgets to control and monitor TC4820 HOT.
+    def __init__(self, name: str) -> None:
+        """Creates the widgets to control and monitor a TC4820.
 
-        Returns:
-            None
+        Args:
+            name (str): Name of the blackbody the TC4820 is controlling
         """
-        super().__init__("TC4820 HOT")
+        super().__init__("TC4820 %s" % name.upper())
```

Comment:
Nowadays in Python it's better to use f-strings:

```suggestion
        super().__init__(f"TC4820 {name.upper()}")
```